build/_static/languages.js

```diff
@@ -0,0 +1,6 @@
+function change_language() {
+    var current_lang = document.documentElement.lang;
+    var new_lang = document.getElementById("language-switcher").value;
+    var url = window.location.href;
+    window.location.href = url.replace(current_lang, new_lang);
```

Comment:
I was testing this locally-- and the "es" in "Desktop" was getting replaced-- the local build was on my Desktop. 
```suggestion
    var regex = new RegExp('/' + current_lang + '(?=/)');
    window.location.href = url.replace(regex, '/' + new_lang);
```

Reply:
Good catch - I will update thanks!